ui/src/app/pages/admin/admin-user.tsx

```diff
@@ -124,6 +112,31 @@ const EmailValidationErrorMessage = ({emailValidationResponse, updatedProfile, v
   return null;
 };
 
+const FreeCreditsUsage = ({aboveLimit, usage}) => {
+  const inputStyle = aboveLimit ?
+  {...styles.textInput,
+    backgroundColor: colorWithWhiteness(colors.danger, .95),
+    borderColor: colors.danger,
+    color: colors.danger,
+  } :
+  {...styles.textInput,
+    ...styles.backgroundColorDark,
+    color: colors.disabled,
+  };
+
+  return <React.Fragment>
+    <TextInputWithLabel
+      labelText={'Free credits used'}
```

Comment:
nit: maybe indicate in the help text or hover  text that  this applies to  all workspaces owned by this user.